lib/modules/manager/mint/extract.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { extractPackageFile } from '.';
+
+const simpleMintfile = Fixtures.get('Mintfile');
+
+const noVersionMintfileContent = `
+yonaskolb/xcodegen
+realm/SwiftLint
+`;
+
+const complexMintFileContent = `
+SwiftGen/SwiftGen@6.6.1
+yonaskolb/xcodegen
+realm/SwiftLint @ 0.48.0`;
+
+const includesCommentOutMintFileContent = `
```

Comment:
```suggestion
const includesCommentedOutMintFileContent = `
```

May be good to rename this `const`? You should do it in VSCode or a similar editor, so you change all instances.